internal/scorecard/testpod.go

```diff
@@ -121,6 +121,94 @@ func getPodDefinition(configMapName string, test v1alpha3.TestConfiguration, r P
 	}
 }
 
+//Creates the same pod definition as above without added security context for legacy users
+func getLegacyPodDefinition(configMapName string, test v1alpha3.TestConfiguration, r PodTestRunner) *v1.Pod {
```

Comment:
Shows as dead code. Do we still need this?

Reply:
Looks like this function is still lingering around. The `sanity` action is saying this code is unused - do we need this?

Reply:
removed code.